atomix/src/test/java/com/atomix/exampletest/ClientExampleTest.java

```diff
@@ -0,0 +1,38 @@
+package com.atomix.exampletest;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertSame;
+import io.atomix.AtomixClient;
+import io.atomix.catalyst.transport.Address;
+import io.atomix.catalyst.transport.netty.NettyTransport;
+import io.atomix.collections.DistributedMap;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.Test;
+
+public class ClientExampleTest {
+
+    @Test
+    public void ExampleTest() throws InterruptedException {
+        AtomixClient client = AtomixClient.builder()
+            .withTransport(new NettyTransport())
+            .build();
+
+        List<Address> cluster = Arrays.asList(new Address("localhost", 8700), new Address("localhsot", 8701));
+        client.connect(cluster)
+            .thenRun(() -> System.out.println("Client Connected"));
+
+        Thread.sleep(5000);
```

Comment:
Can't we replace this with Awaitility or any other form of testing things that happen asynchronously?

Reply:
I did forced synchronization here so that I could make sure client is connected to the cluster and I can retrieve the map value from there in the following line.